_posts/2018-10-21-Integrating-EKS-with-other-AWS-services.md

```diff
@@ -0,0 +1,794 @@
+---
+layout: post
+title: Integrating EKS with other AWS Services
+banner: /assets/posts/2018-10-21-Integrating-EKS-with-other-AWS-services/banner.png
+author: lvandonkersgoed
+
+---
+EKS offers developers an easy way to run Kubernetes workloads at AWS. But what if you need to integrate your EKS based app with other services like CloudFront, API Gateway or Web Application Firewall?
+
+For a recent project we needed to deploy a number of environments with EKS clusters. Each of these environments had different requirements regarding the supporting services; some needed API Gateway, some needed WAF, others needed CloudFront, and many required a combination of these services. We've worked to create a "one size fits all" deployment for EKS - a configuration that allows for integration with all of these services. You can find our results below.
+
+Re:Invent 2018 is being held in Las Vegas next week. We expect that AWS will announce more suported regions for EKS, leading to more EKS deployments in the near future. We hope this blog post will help you set up the integrations and complex configurations for all your use cases.
+
+The challenges we will be discussing fall into two categories:
+- Reliably integrating HTTP frontend services (CloudFront, API Gateway and WAF) with EKS
+- Using Infrastructure as Code (CloudFormation) to deploy resources that depend on EKS managed resources
+
+## About EKS
+Elastic Container Service for Kubernetes (somehow abbreviated to EKS) is Amazon's implementation of a managed Kubernetes service. In a nutshell, Amazon provides the master nodes, you provide the worker nodes, you do some configuration, and voilà: you have a highly available, scalable, and relatively cheap Kubernetes cluster. 
```

Comment:
You mention EKS above but you only explain here the abbreviation. Typically this is done in the first occurrence in the text.